environment-clean-build.yml

```diff
@@ -27,6 +27,3 @@ dependencies:
 - conda-forge::gdown
 - conda-forge::python-wget
 - requests
```

Comment:
I think this environment still needs to include pip until pymovebank is moved to conda.